Makefile

```diff
@@ -31,7 +31,7 @@ else
 DO=eval
 endif
 
-all: docker
+all: manifests docker
```

Comment:
I think it would be better to have the docker target depend on manifests

Reply:
> I think it would be better to have the docker target depend on manifests

I agree, but oddly enough when I tried that here:  https://github.com/kubevirt/containerized-data-importer/pull/883 it "broke all the things".  Need to look into that and understand why; @annastopel mentioned something about env variables not being set, I didn't investigate yet.

Reply:
We do want to be able to call make doker and make manifests in separate commands, to make that docker registry trick we have. 

```
QUAY_NAMESPACE=$QUAY_NAMESPACE DOCKER_PREFIX=$MANIFEST_REGISTRY PULL_POLICY=$(getTestPullPolicy) make manifests
DOCKER_PREFIX=$DOCKER_PREFIX make docker push
  
```

Reply:
That is why I intend to set manifests on all target. It is not clear to me though why there is a failure on travis on ``` make goveralls```

Reply:
the error is 
``` Ran 4 of 4 Specs in 0.000 seconds
SUCCESS! -- 4 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestClonerTarget (0.00s)
PASS
coverage: 56.0% of statements
ok  	kubevirt.io/containerized-data-importer/pkg/util/prometheus	0.011s	coverage: 56.0% of statements
?   	kubevirt.io/containerized-data-importer/pkg/version	[no test files]
?   	kubevirt.io/containerized-data-importer/pkg/version/verflag	[no test files]
make: *** [goveralls] Error 1
The command "make goveralls" exited with 2.
```